n2y/test.py

```diff
@@ -0,0 +1,55 @@
+import logging
```

Comment:
It looks like it's a test script you're using locally. I'd suggest not committing these sorts of files to the repo, as this sort of testing should really be handled in the automated tests.

Reply:
Absolutely.  Next time I will upload this as a separate file in another fashion  if I want you to see it.  I was unable to get the plugin loaded in the proper course of n2y, so I had to write this module to test it.  

test.py is **not** a test module, never mind the name.  It is a work around for my being unable to load as a plugin my plugin.

Reply:
That makes sense! I think once you inherit from `n2y.blocks.LinkToPageBlock` that you'll be able to load your plugin.